lib/matplotlib/axes/_axes.py

```diff
@@ -111,7 +111,8 @@ class Axes(_AxesBase):
     aname = 'Axes'
 
     def get_title(self, loc="center"):
-        """Get an axes title.
+        """
+        Get an axes title.
```

Comment:
@anntzer keeps doing the opposite. Personally, I prefer this form, but we should get some consensus to stop unnecessary churn.

Reply:
Does numpydoc have an opinion about this and does either form play badly with the `autosummary` tables?

Reply:
I got the "same line" form from the example at the bottom of https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings although tbh the text itself says "The summary line may be on the same line as the opening quotes or on the next line.".

Almost certain that any conformant tool should be able to handle *both* forms properly because it should pass docstrings through inspect.cleandoc, which does
> All leading whitespace is removed from the first line. Any leading whitespace that can be uniformly removed **from the second line onwards** is removed. Empty lines at the beginning and end are subsequently removed. Also, all tabs are expanded to spaces.

(emphasis mine).

I think starting on the second line is fine too (and I tend to do that when it buys me three characters for the first line of the docstring...), happy to stick to that format.

Reply:
I prefer the second-line form.